x/proof/types/params.go

```diff
@@ -22,6 +22,9 @@ var (
 	DefaultRelayDifficultyTargetHashHex = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff" // all relays are payable
 	DefaultRelayDifficultyTargetHash, _ = hex.DecodeString(DefaultRelayDifficultyTargetHashHex)
 
+	// TODO_BETA(@red-0ne): Iterate on the parameters below by adding unit suffixes,
+	// whether proof requirement threshold should be in function of the stake amount.
```

Comment:
The first sentence LGTM, but I don't understand what you're intending to say with the second one here.